lib/Github/Api/AbstractApi.php

```diff
@@ -20,6 +20,13 @@
     protected $client;
 
     /**
+     * The requested page (GitHub pagination).
+     *
+     * @var null|int
+     */
+    protected $page;
```

Comment:
I would make $perPage private to if it didn't break BC. 

It is easier to change from private to protected than the other way around.